great_expectations/rule_based_profiler/parameter_builder/unexpected_count_statistics_multi_batch_parameter_builder.py

```diff
@@ -264,6 +262,35 @@ def _build_parameters(  # PLR0915, PLR0915
         )
 
 
+def _standardize_mostly_for_single_batch(  # noqa: PLR0911
+    expectation_type: str, mostly: np.float64
+) -> np.float64:
+    """
+    Applies business logic to standardize "mostly" value for single-Batch case.
+    """
+    if expectation_type == "expect_column_values_to_be_null":
+        if mostly >= 1.0:  # noqa:  PLR0915
+            return np.float64(1.0)
+
+        if mostly >= 0.99:  # noqa:  PLR0915
+            return np.float64(0.99)
+
+        if mostly >= 0.975:  # noqa:  PLR0915
+            return np.float64(0.975)
+
+        return mostly
```

Comment:
@TrangPham This may be a bug: The logic says that if mostly is between `0.975` and `0.99` then we emit `expect_column_values_to_not_be_null` with rounding:
```
`expect_column_values_to_not_be_null()` where mostly is the percent missing rounded down to the nearest 2.5% (e.g. 62% missing gets a mostly of .6, 64% gets a mostly of .625)
```
I believe that @donaldheppner is correct in his earlier comment.  Happy to discuss.  Thank you.